src/JsonSchema/Data/AllGangsSchema.ts

```diff
@@ -1,7 +1,15 @@
-import { JSONSchemaType } from "ajv";
+import type { JSONSchemaType } from "ajv";
 import type { AllGangs } from "../../Gang/AllGangs";
-import { FactionName } from "@enums";
+import { GangConstants } from "../../Gang/data/Constants";
 
+/**
+ * If we add/remove gangs, we must change 3 places:
+ * - src\Gang\AllGangs.ts: getDefaultAllGangs
+ * - src\Gang\data\Constants.ts: GangConstants.Names
+ * - src\Gang\data\power.ts: PowerMultiplier
+ *
```

Comment:
I would also add "save migration code" to this list (doesn't need to be an exact filename, since the exact file might change), because due to these constraints a coordination is *required* if we ever decide to add/remove gangs.